tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -996,7 +996,25 @@ handle_build() {
 
 		echo "${ARTEFACT_REGISTRY_PASSWORD}" | sudo oras login "${ARTEFACT_REGISTRY}" -u "${ARTEFACT_REGISTRY_USERNAME}" --password-stdin
 
-		sudo oras push ${ARTEFACT_REGISTRY}/kata-containers/cached-artefacts/${build_target}:latest-${TARGET_BRANCH}-$(uname -m) ${final_tarball_name} ${build_target}-version ${build_target}-builder-image-version ${build_target}-sha256sum
+		case ${build_target} in
+			kernel*-confidential|kernel-sev)
+				sudo oras push \
+					${ARTEFACT_REGISTRY}/kata-containers/cached-artefacts/${build_target}:latest-${TARGET_BRANCH}-$(uname -m) \
+					${final_tarball_name} \
+					"kata-static-${build_target}-modules.tar.xz" \
+					${build_target}-version \
+					${build_target}-builder-image-version \
+					{build_target}-sha256sum
+				;;
+			*)
+				sudo oras push \
+					${ARTEFACT_REGISTRY}/kata-containers/cached-artefacts/${build_target}:latest-${TARGET_BRANCH}-$(uname -m) \
+					${final_tarball_name} \
+					${build_target}-version \
+					${build_target}-builder-image-version \
+					{build_target}-sha256sum
+				;;
+		esac
```

Comment:
Another suggestion would be  to try removing this duplication. Something like this (if possible):

```bash
additional_arg=""
case ${build_target} in
    kernel*-confidential|kernel-sev)
        additional_arg="kata-static-${build_target}-modules.tar.xz"
        ;;
esac

sudo oras push \
    ${ARTEFACT_REGISTRY}/kata-containers/cached-artefacts/${build_target}:latest-${TARGET_BRANCH}-$(uname -m) \
    ${final_tarball_name} \
    ${additional_arg:+$additional_arg} \
    ${build_target}-version \
    ${build_target}-builder-image-version \
    ${build_target}-sha256sum
```

I haven´t tested this code, so not sure if will work as it is.

Reply:
I'll do this in a follow-up, just to make sure things work first with the "dumber" version.